README.rst

```diff
@@ -22,6 +22,7 @@ functions enabled, use ::
 
    ipython --ext=diffpy.pdffit2.ipy_ext --pylab
 
+
```

Comment:
I don't think we need a new line here.  What was your thinking here?

Reply:
Fixing